### PR TITLE
LXD Host arch

### PR DESCRIPTION
**Pull request by SimonRichardson**
## Description of change

HostArch from the server
The following ensures that the HostArch is from the LXD server and not from the local machine.

## QA steps

Bootstrap to different host architectures when bootstrapping.

## Documentation changes

N/A

## Bug reference

N/A

### CODE REVIEW

**Comment by SimonRichardson** [comment]
https://trello.com/c/KwFDQrL9/363-handle-arch-in-lxd-provider

File: provider/lxd/environ_broker.go [hunk 1]
```diff
@@ -302,7 +299,7 @@ func (env *environ) getHardwareCharacteristics(
 	archStr := container.Arch()
 	if archStr == "unknown" || !arch.IsSupportedArch(archStr) {
 		// TODO(ericsnow) This special-case should be improved.
```
Comment (manadart) [inline provider/lxd/environ_broker.go]: Comment can go now.

**Comment by SimonRichardson** [comment]
!!build!!

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**